tests/installation/ipxe_install.pm

```diff
@@ -291,7 +320,7 @@ sub run {
     }
 
     # Print screenshots for ipxe boot process
-    if (get_var('VIRT_AUTOTEST')) {
+    if (get_var('VIRT_AUTOTEST') || is_agama) {
```

Comment:
Why do you mix virtualization test and agama? You completely skipping `set_bootscript_hdd`.